tsconfig.build.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "target": "ES2015",
+        "module": "ES2015",
+        "composite": true,
+        "moduleResolution": "node",
+        "noImplicitAny": true,
+        "noImplicitOverride": true,
+        "noUnusedParameters": true,
+        "noUnusedLocals": true,
+        "strictNullChecks": true,
+        "strict": true
+    },
+    "references": [
+        { "path": "packages/matter.js" },
+        { "path": "packages/node-matter" }
+    ]
```

Comment:
If there's a clear "entrypoint" to the monorepo, one consideration would be to have no references at the top level, and build in the "entrypoint" package instead.

Reply:
Because of the fact that we really publish two libraryies and none is the real "entrypoint" I would keep this